pact/coin-contract/v2/coin.repl

```diff
@@ -813,3 +831,44 @@
   "gas-guard succeeds when GAS not present, but keyset is"
   true
   (gas-guard (keyset-ref-guard "bez")))
+
+(commit-tx)
+
+(begin-tx)
+
+(use coin)
+
+(expect-failure
+  "Remediations fail without the presence of REMEDIATE"
+  "not granted"
+  (remediate "brandon" 1.0))
+
+(env-keys ["brandon"])
```

Comment:
This is precisely what we _cannot_ do in remediations.

Reply:
ahhhhh `DEBIT` enforces the guard. Right. We'll need to do our own debiting operation